docusaurus-plugin-moonwave/src/index.js

```diff
@@ -86,6 +94,77 @@ function parseClassOrder(content, classOrder, nameSet) {
   }
 }
 
+function parseSimpleApiCategories(luaClass, apiCategories) {
+  const tagSet = new Set(
+    luaClass.functions.filter((func) => func.tags).flatMap((func) => func.tags)
+  )
+
+  let listedCategories = []
+  apiCategories.forEach((category) => {
+    if (tagSet.has(category)) {
+      listedCategories.push({
+        value: capitalize(category),
+        id: category,
+        children: luaClass.functions
+          .filter((func) => func.tags && func.tags.includes(category))
+          .map((member) => {
+            return {
+              value: addFunctionTypeSymbol(member.name, member.function_type),
+              id: member.name,
+              children: [],
+            }
+          })
+          .sort((childA, childB) => childA.id.localeCompare(childB.id)),
+      })
+    }
+  })
```

Comment:
Can we just use a regular `for..of` loop here? We can then use `continue` to bail out early, instead of adding more indentation